velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1132,46 +1132,48 @@ struct DateFormatFunction : public TimestampWithTimezoneSupport<T> {
 
   const date::time_zone* sessionTimeZone_ = nullptr;
   std::shared_ptr<DateTimeFormatter> mysqlDateTime_;
+  uint32_t maxResultSize_;
```

Comment:
Make these private and move after all methods.

Reply:
@PHILO-HE Would you address these? Otherwise, looks great.

Reply:
@mbasmanova, I forgot this fix. Just updated in latest commit. Thanks!